extensions/ipynb/package.json

```diff
@@ -64,8 +64,7 @@
     "watch": "npx gulp watch-extension:ipynb"
   },
 	"dependencies": {
-		"@enonic/fnv-plus": "^1.3.0",
-		"detect-indent": "^6.0.0"
```

Comment:
Removed this npm package & detecting indents ourselves

Reply:
🎉